text/0000-gs1-subteam.md

```diff
@@ -0,0 +1,89 @@
+- Feature Name: gs1_subteam
+- Start Date: 2019-05-04
+- RFC PR:
+- Hyperledger Grid Issue:
+
+# Summary
+[summary]: #summary
+
+Creates a new GS1 subteam to help shepherd RFCs related to the GS1 standards
+through the RFC process. Topics appropriate for this subteam include any
+related to the GS1 standards. This team is reponsible for ensuring that Grid's
+GS1 support is consistent with the GS1 standards and the intent behind the
+those standards.
+
+# Motivation
+[motivation]: #motivation
+
+[Grid RFC 0008](https://github.com/hyperledger/grid-rfcs/blob/master/text/0008-grid-governance.md)
```

Comment:
This was PR/RFC 0008 but the file ended up being named 0000.  We likely need to rename the grid governance file.

Reply:
Handled via #10